packages/client/src/components/NavBar/MobileNav.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import MenuIcon from '@material-ui/icons/Menu';
+import { COLORS } from '../../constants';
+
+const MobileMenuIconWrapper = styled.div`
+  position: absolute;
+  top: 15px;
```

Comment:
We should decide between using multiples of 8 or 5 for consistent spacing: https://blog.prototypr.io/the-8pt-grid-consistent-spacing-in-ui-design-with-sketch-577e4f0fd520